x/claim/types/codec.go

```diff
@@ -9,8 +9,8 @@ import (
 )
 
 func RegisterCodec(cdc *codec.LegacyAmino) {
-	cdc.RegisterConcrete(&MsgSetAirdropAllocations{}, "claim/SetAirdropAllocation", nil)
-	cdc.RegisterConcrete(&MsgClaimFreeAmount{}, "claim/ClaimFreeAmount", nil)
+	cdc.RegisterConcrete(&MsgSetAirdropAllocations{}, "claim/MsgSetAirdropAllocations", nil)
+	cdc.RegisterConcrete(&MsgClaimFreeAmount{}, "claim/MsgClaimFreeAmount", nil)
 	cdc.RegisterConcrete(&MsgCreateAirdrop{}, "claim/MsgCreateAirdrop", nil)
 	cdc.RegisterConcrete(&MsgDeleteAirdrop{}, "claim/MsgDeleteAirdrop", nil)
```

Comment:
can you use `legacy.RegisterAminoMsg` for all these so it validates the length?

Reply:
7711a8c3d5eabce5a104014f9595321eeed67fd0